Gems/LyShine/Assets/Passes/LyShinePassRequest.azasset

```diff
@@ -12,9 +16,10 @@
             {
                 "LocalSlot": "DepthInputOutput",
                 "AttachmentRef": {
-                    "Pass": "DepthPrePass",
+                    "Pass": "HairParentPass",
```

Comment:
Seems odd to be referencing hair directly in LyShine - is this correct?

Reply:
It's a issue. I will fix it